docs/source/issue/view.rst

```diff
@@ -32,11 +32,11 @@ Options
 
 ::
 
-  -c, --comments       Show mr comments and activities
+  -c, --comments       Show issue comments and activities
```

Comment:
No need to change the docs here. They are generated automatically when creating a release using the `make gen-docs` command.